services/network_config_manager/pom.xml

```diff
@@ -49,11 +49,27 @@ Copyright(c) 2020 Futurewei Cloud
     </repositories>
 
     <dependencies>
```

Comment:
@zzxgzgz A new file called executionHistory.lock is added by this PR. This may be a local file. 

Please check and remove it if so.

Reply:
Removed